src/main/java/net/discordjug/javabot/systems/staff_commands/forms/model/FormField.java

```diff
@@ -0,0 +1,95 @@
+package net.discordjug.javabot.systems.staff_commands.forms.model;
+
+import net.dv8tion.jda.api.interactions.components.text.TextInput;
+import net.dv8tion.jda.api.interactions.components.text.TextInputStyle;
+
+/**
+ * Represents a form field.
+ */
+public class FormField {
+	private final String label;
+	private final int max;
+	private final int min;
+	private final String placeholder;
+	private final boolean required;
+	private final String style;
```

Comment:
It would also be possible to use the `TextInputStyle` annotation here and then use methods like `.name()`/`valueOf()`